homeassistant/components/sensor/fixer.py

```diff
@@ -0,0 +1,120 @@
+"""
+Currency exchange rate support that comes from fixer.io.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.fixer/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.const import (CONF_PLATFORM, CONF_NAME)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['fixerio==0.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "Exchange rate"
+ICON = 'mdi:currency'
+
+CONF_BASE = 'base'
+CONF_TARGET = 'target'
+
+STATE_ATTR_BASE = 'Base currency'
+STATE_ATTR_TARGET = 'Target currency'
+STATE_ATTR_EXCHANGE_RATE = 'Exchange rate'
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): 'fixer',
+    vol.Optional(CONF_BASE): cv.string,
+    vol.Optional(CONF_NAME): cv.string,
+    vol.Required(CONF_TARGET): cv.string,
+})
+
+# Return cached results if last scan was less then this time ago.
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=3600)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Fixer.io sensor."""
+    from fixerio import (Fixerio, exceptions)
+
+    name = config.get(CONF_NAME, DEFAULT_NAME)
+    base = config.get(CONF_BASE, 'USD')
+    target = config.get(CONF_TARGET)
+
+    try:
+        Fixerio(base=base, symbols=[target], secure=True).latest()
+    except exceptions.FixerioException:
+        _LOGGER.error('One of the given currencies is not supported')
+        return False
+
+    data = ExchangeData(base, target)
+    add_devices([ExchangeRateSensor(data, name, target)])
+
+
+# pylint: disable=too-few-public-methods
+class ExchangeRateSensor(Entity):
```

Comment:
Shouldn't `unit_of_measurement` be `self._target` ? That way you can get a graph